bundles/binding/org.openhab.binding.bticino/src/main/java/org/openhab/binding/bticino/internal/BticinoDevice.java

```diff
@@ -174,12 +213,155 @@ public void receiveCommand(String itemName, Command command, BticinoBindingConfi
+                                                        (int) (RollerShutterPosStart - RollerShutterPosEnd));
+                                            }
+                                            if (RollerShutterDir.getOrDefault(l_where, "") == "DOWN") {
+                                                logger.debug("RollerShutterTimeStart: {}: {}", itemName,
+                                                        ((new Date()).getTime() - RollerShutterTimeStart
+                                                                .getOrDefault(l_where, new Date((new Date()).getTime()))
+                                                                .getTime()));
+                                                Long RollerShutterPosEnd = ((new Date()).getTime()
+                                                        - RollerShutterTimeStart
+                                                                .getOrDefault(l_where, new Date((new Date()).getTime()))
+                                                                .getTime())
+                                                        * 100 / m_ShutterRunTime;
+                                                RollerShutterPos.put(l_where,
+                                                        (int) (RollerShutterPosStart + RollerShutterPosEnd));
+                                            }
+                                        }
+                                        RollerShutterDir.put(l_where, "");
+                                        if (RollerShutterPos.get(l_where) > 100) {
+                                            RollerShutterPos.put(l_where, 100);
+                                        }
+                                        if (RollerShutterPos.get(l_where) < 0) {
+                                            RollerShutterPos.put(l_where, 0);
+                                        }
+                                        logger.debug("RollershutterPosEnd: {}: {}", itemName,
+                                                RollerShutterPos.get(l_where));
+                                        eventPublisher.postUpdate(itemName,
+                                                PercentType.valueOf(Integer.toString(RollerShutterPos.get(l_where))));
+                                        return;
+                                    }
+                                }
+                            }
+                        }
+
+                    }
+                    // Temperature Control
+                    case 4: {
+                        // Set Temperature Heating Zone
+                        // *#4*#WHERE*#14*T*M##
+                        // WHERE = #1-99
+                        // T = Temperature 0400 (40 degrees C)
+                        // M = Mode (1=Heating, 2=Cooling 3=General)
+                        // #0 is used in the WHERE for ALL ZONES
+                        if (itemBindingConfig.what.equalsIgnoreCase("14")) {
```

Comment:
No point in using `equalsIgnoreCase()` here.

Reply:
OK
see commit https://github.com/openhab/openhab1-addons/pull/5486/commits/c1c8f3a86ba8641c7333467de21a0ba489322073